### PR TITLE
chore: allow disabling per tokenizer

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| Fixes RM-541 |
|:---:|

## 🧰 Changes

Allow fine grain control over which tokenizers are in use. While developing for @readme/editor, we'd like to keep the editor non-lossy. The easiest way to do that is to disable tokenizers that we don't yet handle.

If we don't disable a tokenizer, we could convert it back to plain text. But it gets complicated, because we'd have to know what text was 'eaten' by the markdown engine.

### CODE REVIEW

File: docs/tables.md [hunk 1]
```diff
@@ -54,8 +54,22 @@ Tables have been simplified to mirror a more standard implementation. We've also
...
+    --table-edges: rgba(34, 5, 64, .5);
+    --table-row: white;
+  }
+
+  #rdmd-demo .markdown-body .rdmd-table thead tr {
+    box-shadow: none;
+  }
+
+  #rdmd-demo .markdown-body .rdmd-table thead tr th:last-child {
+    box-shadow: none;
```
Comment (kellyjosephprice) [inline docs/tables.md:73]: Something changed with the hubs css? I didn't spend any more than a second looking at it. Instead, I just got rid of the drop shadow, which removes the border radius of the table. :sadpanda;

File: __tests__/disabling-tokenizers.test.js [hunk 2]
```diff
@@ -27,3 +27,17 @@ describe('disableTokenizers: "blocks"', () => {
...
+    const md = '# heading 1';
+    expect(markdown.mdast(md, opts)).toMatchSnapshot();
+  });
+
+  it('disables an inline tokenizer', () => {
+    const opts = { disableTokenizers: { inline: ['code'] } };
+    const md = '`const js = true`';
+    expect(markdown.mdast(md, opts)).toMatchSnapshot();
+  });
+});
```
Comment (rafegoldberg) [inline __tests__/disabling-tokenizers.test.js:43]: ### 🦾

**Review by rafegoldberg — APPROVED** [review]
Looks good to me!

File: docs/tables.md (hunk 1 above)
Comment (rafegoldberg) [reply docs/tables.md:73]: Sounds like it could also be the changes from #108 causing this issue?

Comment (kellyjosephprice) [reply docs/tables.md:73]: Oh right.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.26.3-next.1</b>](/readmeio/markdown/releases/tag/v6.26.3-next.1)
<br>

**Pull request merged**